src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+
+for (const row of table.rows) {
+  const cellToClone = row.cells[1];
+  const clonedCell = cellToClone.cloneNode(true);
+  const lastCellIndex = row.cells.length - 1;
+
+  row.insertBefore(clonedCell, row.cells[lastCellIndex]);
```

Comment:
The code inserts the cloned cell before the last cell, which means the cloned cell will be placed as the second-to-last cell. If the intention is to place it at the end, consider using `row.appendChild(clonedCell);` instead. If the intention is to place it before the last cell, the current implementation is correct.